src/Stack/Ide.hs

```diff
@@ -43,7 +43,7 @@ ide
     -> [String] -- ^ GHC options.
     -> m ()
 ide targets useropts = do
-    (_realTargets,_,pkgs) <- ghciSetup Nothing targets
+    (_realTargets,_,pkgs) <- ghciSetup (Just BSAll) Nothing targets
```

Comment:
Actually, I think the right behavior for stack ide is to only build dependencies.